python/GafferUI/NumericWidget.py

```diff
@@ -75,7 +76,14 @@ def setValue( self, value ) :
 
 	def getValue( self ) :
 
-		return self.__numericType( self.getText() )
+		value = self.getText()
+
+		# Allow derived classes to modify the validator if required
```

Comment:
I don't understand this comment. What is the expected use case, and if someone does provide a different validator, why do we then ignore it with our `isinstance()` check?

Reply:
Simplified [here](https://github.com/GafferHQ/gaffer/pull/3873/commits/87cdd4dc907ebdf117de10ef33aec0c3e5421ddf)